stacks/api/function/horizon/test/horizon.spec.ts

```diff
@@ -66,19 +67,20 @@ describe("horizon", () => {
   let horizon: Horizon;
   let app: INestApplication;
   let spawnSpy: jasmine.Spy;
+  let horizonOptions = {
+    databaseUri: undefined,
+    databaseName: undefined,
+    databaseReplicaSet: undefined,
+    poolSize: 10,
+    publicUrl: undefined,
+    timeout: 60000
+  };
```

Comment:
I couldn't get why you needed such a change.

Reply:
I need to pass the timeout value to this function `clock.tick();`

Reply:
https://github.com/spica-engine/spica/blob/9e01d3c3e20c52d3ef1619465e5aaf2b99b4a433/stacks/api/function/horizon/test/horizon.spec.ts#L155